docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md

```diff
@@ -41,45 +41,52 @@ Complete the steps in one of the following sections to set up an add-in catalog
 
 9. To add an add-in to the Office Add-ins add-in catalog, choose **Apps for Office**.
 
-### To set up an add-in catalog on Office 365
+### To set up an app catalog on Office 365
 
-1. On the Office 365 admin center page, choose  **Admin**, and then choose  **SharePoint**.
+Even though SharePoint names the catalog an "app" catalog, you can register Office Add-ins in the app catalog.
 
-2. In the left task pane, choose  **add-ins**.
+1. Go to the Microsoft 365 admin center. For information on how to find the admin center, see [About the Microsoft 365 admin center](https://docs.microsoft.com/office365/admin/admin-overview/about-the-admin-center)
 
-3. On the  **add-ins** page, choose **Add-in Catalog**.
+2. On the Microsoft 365 admin center page, expand the list of **Admin centers**, and then choose **SharePoint**.
 
-4. On the  **Add-in Catalog Site** page, choose **OK** to accept the default option and create a new add-in catalog site.
+    > [!NOTE]
+    > You need to use the Classic SharePoint admin center to set up the catalog. If you are in the new SharePoint admin center, choose **Class SharePoint admin center** in the left pane.

```

Comment:
**Class SharePoint admin center** <-- is this a typo?  i.e., should it be: **Classic SharePoint admin center**

Reply:
fixed